### ISSUE TITLE
About warmup step of CLIP

### ISSUE THREAD

**Issue by Asthestarsfalll** [lucidrains/big-sleep#138]
I notice that there is a warmup step of CLIP, could you please tell me what the issue is?

```python
with torch.no_grad():
    self.model(self.encoded_texts["max"][0]) # one warmup step due to issue with CLIP and CUDA
```
